R/metadata.R

```diff
@@ -46,10 +54,17 @@ setClass(
     load.rate.units="character",
     # Other
     station="character",
+    wq.sta.id="character",
+    station.lat="ANY",
```

Comment:
should we force `station.lat` to be `"numeric"` when stored in a metadata object? that would make it more readily usable for calculations.

Later, we could get fancy:
* we could accept inputs in character (`"40° 48' 24.7032\" N"`) or numeric (40.843) form in the `metadata()` and `updateMetadata()` functions, then translate everything to numeric within the `updateMetadata()` function in the process of creating/updating the metadata object
* we could print out the coordinates in character form in the metadata method for `show()`